CRM/Core/Form/Task/Batch.php

```diff
@@ -133,7 +133,7 @@ public function buildQuickForm() {
     $this->addButtons([
       [
         'type' => 'submit',
-        'name' => ts('Update ' . ucfirst($this::$entityShortname) . 's)'),
+        'name' => ts('Update'),
```

Comment:
So this changes it from eg. "Update Membership" to "Update" - I think that's ok

Reply:
yep, fixing this was too complicated, and it seemed like the context was sufficient to understand what is being updated.